### ISSUE TITLE
[BUG] Duplicate Success Message on Artifact Update

### ISSUE THREAD

**Issue by epamLDadayan** [ProjectAlita/projectalita.github.io#477]
Description: When a user makes changes and saves an artifact (prompt, datasource, or agent), the success message indicating that the save was successful is displayed twice. The message should only be shown once.

Steps to Reproduce:

1. Navigate to an artifact (prompt, datasource, or agent) page.
2. Make changes to the artifact.
3. Save the changes.
4. Observe the success message.

Expected Result: The success message should be displayed only once.

Actual Result: The success message is displayed twice.

Additional Information:

- Issue: Duplicate success message on artifact save.
- Reproducibility: Consistently reproducible when saving any artifact.
- Artifacts Affected: Prompts, datasources, agents.

https://github.com/user-attachments/assets/5315badf-f223-45be-8090-0fc9fd72421c

**Comment by epamLDadayan** [comment]
@HawkQing1 , it causes another issue no update message displayed for Agents. For prompts and Datasources, it is fixed.

**Comment by HawkQing1** [comment]
@epamLDadayan. Please verify it on main

**Comment by epamLDadayan** [comment]
Fixed, Tested on Main env.
